cpp/test/distance/distance_base.cuh

```diff
@@ -450,10 +449,6 @@ class DistanceTest : public ::testing::TestWithParam<DistanceInputs<DataType>> {
     }
     naiveDistance(
       dist_ref.data(), x.data(), y.data(), m, n, k, distanceType, isRowMajor, metric_arg, stream);
-    //    size_t worksize = raft::distance::getWorkspaceSize<distanceType, DataType, DataType,
```

Comment:
If this is unneeded, can it just be removed?

Reply:
this is commented code which I've already removed.

Reply:
Got it! Wow, I totally misread the diff.